docs/team/alanlowzies.md

```diff
@@ -1,6 +1,103 @@
-# Alan - Project Portfolio Page
+# Project Portfolio Page of Alan Low (alanlowzies)
 
 ## Overview
+This page showcases my contributions to the development of WerkIt!, a team project (tP) in the CS2113T 
+Software Engineering & Object-Oriented Programming module offered by the School of Computing, National University of 
+Singapore.
 
+### About the Project
+WerkIt! is a command line interface (CLI) application written in Java hat allows users to create a weekly workout 
+schedule for them to refer to and follow. More details about the project can be found in the following locations:
+* [GitHub Repository](../../)
+* [WerkIt! User Guide](../UserGuide.md)
+* [WerkIt! Developer Guide](../DeveloperGuide.md)
 
-### Summary of Contributions
+---
+
+## Summary of Contributions
+### Code Contributed
+A detailed report of my code contributions to the tP can be found in the [tP Code Dashboard](https://nus-cs2113-ay2122s2.github.io/tp-dashboard/?search=alanlowzies&breakdown=true) 
+hosted by the module coordinators of CS2113T.
+
+![tP Code Dashboard](../images/ppp/alanlowzies/tPCodeDashboard.png)
+
+A summary of my code contributions are as follows:
+- Wrote most of the `FileManager` API, which allows WerkIt! to read and write application data
+into text files stored on the user's local filesystem 
+([[WIP] Developer Guide]()). 
```

Comment:
Line 27: is this supposed to be the correct link? it seems to link to the ppp folder for me?

Reply:
Yeah the link is still WIP, I intend to link to the File Manager section in our DG when I've written it 😬